### PR TITLE
Fix duplicate use_system_scm option in #17832

### PR DESCRIPTION
**Pull request by plu**

### CODE REVIEW

**Comment by plu** [comment]
@googlebot I signed it!

**Comment by plu** [comment]
Fixes duplicate option `use_system_scm` in another PR: #17832

**Review by ainame — APPROVED** [review]
Nice fix👍 
I would leave this PR to @rogerluan to merge.

File: fastlane.gemspec [hunk 1]
```diff
@@ -102,7 +102,7 @@ Gem::Specification.new do |spec|
 
   # Development only
   spec.add_development_dependency('rake')
-  spec.add_development_dependency('rspec', '~> 3.9.0')
+  spec.add_development_dependency('rspec', '~> 3.10.0')
```
Comment (ainame) [inline fastlane.gemspec:105]: Do you actually need this bump in "this PR"? If so can you please clarify that?

Comment (plu) [reply fastlane.gemspec:105]: Yes, to get the `include(...).once` matched.

**Comment by plu** [comment]
Thank you for the review!

**Review by rogerluan — APPROVED** [review]
This is great! Really nice to use `include(…).once` 🌟 

Thank you so much for opening a PR fixing the issue and adding tests ❤️

**Pull request merged**